themes/default/layouts/migrate/cloudformation.html

```diff
@@ -39,56 +42,49 @@ <h2>Increase productivity with programming languages</h2>
         <div class="md:flex my-8">
             <div class="md:w-1/2 md:mr-4">
                 <h3>Creating a Simple Web Server on AWS</h3>
+                <p>In this example, we use JavaScript to create a simple web server on AWS using EC2.</p>
                 <p>
-                    In this example, we use JavaScript to create a simple web server on AWS
-                    using EC2.
-                </p>
-                <p>
-                    This code creates the necessary security group, deploys a very simple web
-                    server for example purposes, and then creates the instance, before
-                    exporting the IP and hostname.
+                    This code creates the necessary security group, deploys a very simple web server for example purposes, and then creates the instance, before exporting the IP
+                    and hostname.
                 </p>
+                <p>Pulumi can be used on any resource on AWS, Azure, GCP, Kubernetes, and OpenStack, covering serverless, containers, and infrastructure.</p>
                 <p>
-                    Pulumi can be used on any resource on AWS, Azure, GCP, Kubernetes, and
-                    OpenStack, covering serverless, containers, and infrastructure.
-                </p>
-                <p>
-                    <a class="link" href="https://github.com/pulumi/examples" target="_blank">Find many
-                    other examples here.</a>
+                    <a class="link" href="https://github.com/pulumi/examples" target="_blank">Find many other examples here.</a>
                 </p>
             </div>
             <div class="md:w-1/2 md:ml:4">
                 {{ $code := `const aws = require("@pulumi/aws");
 
-let size = "t2.micro";
-let ami  = "ami-7172b611"
+                    let size = "t2.micro";
```

Comment:
This causes weird formatting issues. Not sure if we can ignore it or will need to move any inline code blocks to the front matter.

Reply:
We can def do one-off ignores: https://prettier.io/docs/en/ignore.html.  If you want to flag here areas where you think we should add them, I can do it in this PR, or we can re-address in a follow-up.